mlflow/store/abstract_store.py

```diff
@@ -207,7 +208,7 @@ def search_runs(self, experiment_ids, search_filter, run_view_type, max_results)
         :param search_filter: :py:class`mlflow.utils.search_utils.SearchFilter` object to encode
             search expression or filter string
         :param run_view_type: ACTIVE, DELETED, or ALL runs
-        :param max_results: Maximum number of runs desired
+        :param max_results: Maximum number of runs desired. Default: 1000
```

Comment:
Putting the default here (and in other doc strings) seems brittle since it may change, though it is nice to have it for documentation. Not sure if there is a better way though?

Reply:
Good one. We could reference the constant, using `py:const` but it displays the constant name and `mlflow.store` is not documented.

![image](https://user-images.githubusercontent.com/6900999/56236906-945f7200-603f-11e9-8481-ed000382f192.png)

Also, looks like default arguments are pre-filled with `1000`. So I could go ahead and remove it from the `:param` line in docs.